config/webpack/webpack.shims.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
       // Allow access to PruneClusterForLeaflet PruneCluster from outside the module
       {
         test: require.resolve('prunecluster/dist/PruneCluster'),
-        loader: 'exports-loader?PruneClusterForLeaflet,PruneCluster',
+        loader: 'exports-loader?PruneClusterForLeaflet|PruneCluster',
```

Comment:
Tried if `|` would fix this branch, but nope.

Might need a bit more work: https://github.com/webpack-contrib/exports-loader#options

Reply:
This needs to be moved to `options` in Webpack 5 anyway (https://github.com/Trustroots/trustroots/pull/1793)